recipes/master.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Cookbook Name:: osl-jenkins
+# Recipe:: master
+#
+# Copyright 2014, Oregon State University
```

Comment:
The general rule is to only update the year if you modify the file in that new year.